recipes/m4/all/conanfile.py

```diff
@@ -100,10 +103,15 @@ def build(self):
         autotools.make()
 
     def package(self):
-        copy(self, "COPYING", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
+        if os.path.islink(os.path.join(self.source_folder, "COPYING")):
+            # Use mkdir and copyfile when "COPYING" is a symlink as it is when building with source from Git repo
+            os.mkdir(os.path.join(self.package_folder, "licenses"))
+            shutil.copyfile(os.path.join(self.source_folder, os.readlink(os.path.join(self.source_folder, "COPYING"))),
+                         os.path.join(self.package_folder, "licenses", "COPYING"))
+        else:
+            copy(self, "COPYING", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
```

Comment:
Is is really needed? Please, revert if not. As we are not using Git repo, I don't see symlink as a possibility.

Reply:
Hi @uilianries,
It is only needed when building from the Git source, which is what we'd like to do in case we need to make fixes/customizations. CCI is limiting the applicability of it recipes by not supporting this common workflow. That said I have removed the extra logic per your request. Thanks